### ISSUE TITLE
question - about adding remote server

### ISSUE THREAD

**Issue by moth1995** [juce/fiveserver#1]
Hi Juce, 

I've been using your Fiveserver, first on windows, then i learnd a lot about linux and switch to it, now im running it again since 2 month ago, on docker :) now i got a question, it is possible to add another server instance into the selection of server but not in the same host, i want it on another location, for example, my main server now is running on a VPS on Miami, i got my own server running at my home (Argentina) and i wanna know if it will be possible to point it to this one, my idea will be to have many vps around the globe but make all of them converge into one and only server, i'll post a picture if i wasn't really clear

![idea](https://user-images.githubusercontent.com/36039932/79064908-08e4e500-7c83-11ea-8b15-1f193adea404.png)

I know this maybe is possible, the more complicated thing about this is the DB thinking it clear is kind of hard, but not impossible i think

**Comment by juce** [comment]
yeah, it is probably possible. Should not be difficult to add, i believe the protocol allows for that.
The other server may have its own separate database, i guess.

**Comment by themasterz** [comment]
Hello,
That's works with PES6J, 
https://github.com/pinfake/pes6j
you can configure (ip, name,port,wport) for each servers.
You need same database for all servers.
![servers](https://github.com/juce/fiveserver/assets/12144819/357e4eec-bfc0-4472-ab1e-fef5de090d56)

**Comment by moth1995** [comment]
Closing this, since my question was more related to provide a way to show differents servers of different owners and it will not work since the db must be same, and it makes sense since it was konami owning all servers same db and different servers
